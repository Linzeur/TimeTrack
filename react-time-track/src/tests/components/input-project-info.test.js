import React from "react";
import { render } from "@testing-library/react";

import InputProjectInfo from "../../components/input-project-info";

test("Render close projects", () => {
  const { asFragment } = render(<InputProjectInfo />);
  expect(asFragment()).toMatchSnapshot();
});
