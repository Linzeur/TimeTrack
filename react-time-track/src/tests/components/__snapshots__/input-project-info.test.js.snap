// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Render close projects 1`] = `
<DocumentFragment>
  <section
    class="css-1y69pia-InputProjectInfo"
  >
    <form
      class="css-ntwt0t-InputProjectInfo"
    >
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <label
          class="css-1ktmb7r-InputProjectInfo"
        >
          Client
        </label>
        <div
          class="css-2wl3q5-InputProjectInfo"
        >
          <input
            class="css-rzmpyt-InputProjectInfo"
            name="txtClient"
            required=""
            type="text"
            value=""
          />
        </div>
      </fieldset>
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <label
          class="css-1ktmb7r-InputProjectInfo"
        >
          Project
        </label>
        <div
          class="css-2wl3q5-InputProjectInfo"
        >
          <input
            class="css-rzmpyt-InputProjectInfo"
            name="txtProject"
            required=""
            type="text"
            value=""
          />
        </div>
      </fieldset>
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <label
          class="css-1ktmb7r-InputProjectInfo"
        >
          Category
        </label>
        <div
          class="css-2wl3q5-InputProjectInfo"
        >
          <input
            class="css-rzmpyt-InputProjectInfo"
            name="Category"
            required=""
            type="text"
          />
        </div>
      </fieldset>
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <label
          class="css-1ktmb7r-InputProjectInfo"
        >
          Product
        </label>
        <div
          class="css-2wl3q5-InputProjectInfo"
        >
          <input
            class="css-rzmpyt-InputProjectInfo"
            name="Product"
            required=""
            type="text"
          />
        </div>
      </fieldset>
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <label
          class="css-1ktmb7r-InputProjectInfo"
        >
          Start
        </label>
        <div
          class="css-2wl3q5-InputProjectInfo"
        >
          <input
            class="css-1gyw4au-InputProjectInfo"
            name="txtStart"
            required=""
            type="date"
          />
        </div>
      </fieldset>
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <label
          class="css-1ktmb7r-InputProjectInfo"
        >
          End
        </label>
        <div
          class="css-2wl3q5-InputProjectInfo"
        >
          <input
            class="css-1gyw4au-InputProjectInfo"
            name="txtEnd"
            required=""
            type="date"
          />
        </div>
      </fieldset>
      <fieldset
        class="css-14ljpwi-InputProjectInfo"
      >
        <button
          class="css-frnet4-InputProjectInfo"
          type="button"
        >
          Cancel
        </button>
        <button
          class="css-frnet4-InputProjectInfo"
          type="submit"
        >
          Next
        </button>
      </fieldset>
    </form>
  </section>
</DocumentFragment>
`;
